Extension/src/githubAPI.ts

```diff
@@ -84,7 +87,7 @@ function isArrayOfBuilds(input: any): input is Build[] {
     }
     // Only check the five most recent builds for validity -- no need to check all of them
     for (let i: number = 0; i < 5 && i < input.length; i++) {
-        if (!isBuild(input[i])) {
+        if (!isBuild(input[i], false)) {
```

Comment:
Here, I check if the entries are type build with any number of assets. When upgrading or downgrading, the invalid assets will be filtered out.